### PR TITLE
DT-1364: Add "latest" endpoint that redirects to the latest build

### PR DESCRIPTION
**Pull request by keegan-lillo**
https://sixfivelabs.myjetbrains.com/youtrack/issue/DT-1364

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FILBERT_BUILD_BUCKET=sf-builds
+FILBERT_BUILD_PREFIX=ide/
+FILBERT_CHANNEL_BUCKET=sf-builds
+FILBERT_CHANNEL_PREFIX=ide/
```
Comment (mblakele) [inline .env.sample]: These values aren't exactly secrets but I'd rather not have them in a public git repo

Comment (keegan-lillo) [inline .env.sample]: Oh Sorry. I made the assumption that this was a private repo

**Comment by mblakele** [comment]
LGTM

**Review by marcello3d — APPROVED** [review]
Server changes look good to me. It'd be nice to get CI on the project (maybe travis or circle?). Also need to decide how to deploy with the right version of node.

**Pull request merged**